libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -223,7 +252,7 @@ export class XCCDFResultsMapper extends BaseConverter {
           path: 'Benchmark',
           transformer: (input: Record<string, unknown>): string => {
             const descriptionPaths = [
-              ['description.text', 'description'],
+              ['description.text'],
```

Comment:
Because here, it's the path is used as an "and" not an "or"

The value used here is the combination of the list items (not a fallback, as is the case elsewhere). So if we used `description` and `description.text` here, then the value used would have the description twice, which is not desirable. That's observable by making this change then running the tests - the duplication is evident in the output.